lib/alice/adapters.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Alice.Adapters do
+  @moduledoc false
+
+  def start_link(opts \\ [start_adapter: true]) do
+    if opts[:start_adapter], do: selected_adapter.start_link
+    {:ok, selected_adapter}
+  end
+
+  @doc """
+  Selects adapter, defaults to Slack
+  """
+  def selected_adapter do
+    Application.get_env(:alice, :adapter, Alice.Adapters.Slack)
```

Comment:
👍  Cool, I like what you guys are doing with this. I was curious as I saw that limitation in Hubot, so I am looking into other Bot options. I am not very familiar with Elixir yet.